zk-sdk/src/macros.rs

```diff
@@ -0,0 +1,76 @@
+// Internal macros, used to defined the 'main' impls
+#[macro_export]
```

Comment:
Is the `#[macro_export]` annotation required?

Reply:
Good point! I removed the annotation. Thanks!